File: src/Workflow.Domain/DataObject/Worklists/WorkTypeException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Workflow.Dto.Worklists {

    public class WorkTypeExceptionDto {
        public string Name { get; set; }        
        public string Process { get; set; }
        public string ProcessPath { get; set; }
        public string Activity { get; set; }
        public string ActDisplayName { get; set; }
        public List<string> DestUsers { get; set; }
        public List<DestinationDto> Destinations { get; set; }
    }
}

File: src/Workflow.Web.Service/Workflow/app/model/common/worklists/WorkTypeException.js
Ext.define('Workflow.model.common.worklists.WorkTypeException', {
    extend: 'Ext.data.Model',
    fields: [
		'Name',
		'Id',
		'Process',
        'ProcessPath',
		'Activity',
        'ActDisplayName'
    ],
    hasMany: [
        {
            name: 'Destinations',
            model: 'Workflow.model.common.worklists.Destination',
            associationKey: 'Destinations'
        }
    ]
});